### ISSUE TITLE
Serpent Hold Healer doesn't work

### ISSUE THREAD

**Issue by Fenyx4** [MagerValp/u4remastered#6]
This is a problem in the Apple II and DOS versions as well.

In the DOS version 0x0C on this line needs to be changed to 0x12.
https://github.com/ergonomy-joe/u4-decompiled/blob/c2c2108fa3bb346bcd1d8c207c526f33a4c8f5ef/SRC/U4_TALK.C#L333

I'm guessing C64 version has the same array buried somewhere in it.

Sorry I would dig into it myself but I am neck deep in the DOS version at the moment. :)

**Comment by sean-gugler** [comment]
Yep, it sure does! This is fixed with my pending pull request. See my wiki for other issues resolved: https://github.com/sean-gugler/u4remastered/wiki/Version_2_3_0

That was one of my favorites bugs to discover. It's so human to confuse dec(12) with hex(12)! This kind of mistake is much less likely to occur anymore with the sophisticated tools that now exist, so it's a reminder of how much loving attention went into hand-crafting these games back then.
